src/com/amazon/ion/impl/lite/IonStructLite.java

```diff
@@ -101,6 +101,14 @@ private void build_field_map()
             _field_map.put(v._fieldName, ii); // this causes the map to have the largest index value stored
         }
     }
+
+    @Override
+    public void makeReadOnly() {
```

Comment:
Is there a reason to only do this for `IonStructLite` as opposed to `IonContainerLite`?

Reply:
Do what, specifically? The field map is contained in `IonStructLite`.

Reply:
I was wondering if eager initialization is needed for `IonContainerLite` as well, but it seems the issue is isolated to `IonStructLite`